uniffi_macros/src/export/attributes.rs

```diff
@@ -86,6 +89,15 @@ impl UniffiAttributeArgs for ExportFnArgs {
             let _: Token![=] = input.parse()?;
             Ok(Self {
                 async_runtime: Some(input.parse()?),
+                name: None,
+            })
+        } else if lookahead.peek(kw::name) {
+            let _: kw::name = input.parse()?;
+            let _: Token![=] = input.parse()?;
+            let name = Some(ident_to_string(&input.parse::<Ident>()?));
```

Comment:
`.parse::<Ident>()` will forbid Rust keywords in the input, which seems wrong, I could see one primary use for this feature being when you wanna use a Rust keyword as the foreign language name. `Ident::parse_any` gets around that.

Reply:
As above, I'll change to a string, but I think we [already support Rust keywords](https://github.com/mozilla/uniffi-rs/tree/main/fixtures/keywords/rust/src) as foreign language names.

Reply:
Yeah, supporting it elsewhere is why it should not cause a parse error here either. But moot point anyways, as you noted.